tests/publiccloud/run_ltp.pm

```diff
@@ -65,13 +65,13 @@ sub upload_ltp_logs
         my $ltp_log = Mojo::JSON::decode_json($log_file->slurp());
         my $parser = OpenQA::Parser::Format::LTP->new()->load($log_file->to_string);
         my %ltp_log_results = map { $_->{test_fqn} => $_->{test} } @{$ltp_log->{results}};
-        my $whitelist = LTP::WhiteList->new();
+        #my $whitelist = LTP::WhiteList->new();
 
-        for my $result (@{$parser->results()}) {
-            if ($whitelist->override_known_failures($self, {%{$self->{ltp_env}}, retval => $ltp_log_results{$result->{test_fqn}}->{retval}}, $ltp_testsuite, $result->{test_fqn})) {
-                $result->{result} = 'softfail';
-            }
-        }
+        # for my $result (@{$parser->results()}) {
```

Comment:
Hi , sorry not clear to me why you deleted all this code, may be some misunderstnding ?
Let we discuss a bit of it.

Reply:
Also, as general tip, you could rename your PR with a leading `**Draft**: `, until you see your code ready, to avoid automatic but unexpexted merge.